### PR TITLE
fix: fix uuid setup in PostgreSQL

### PR DESCRIPTION
**Pull request by agnes512**
## Checklist

👉 [Read and sign the CLA (Contributor License Agreement)](https://cla.strongloop.com/agreements/strongloop/loopback-connector-postgresql) 👈

- [ ] `npm test` passes on your machine
- [ ] New tests added or existing tests modified to cover all changes
- [ ] Code conforms with the [style guide](https://loopback.io/doc/en/contrib/style-guide-es6.html)
- [ ] Commit messages are following our [guidelines](https://loopback.io/doc/en/contrib/git-commit-messages.html)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -737,10 +737,25 @@ Auto-migrate supports the automatic generation of property values for the id pro
 }
 ```
 
-It is common to use UUIDs as the primary key in PostgreSQL instead of integers. You can enable it with the following settings:
+It is common to use UUIDs as the primary key in PostgreSQL instead of integers. You can enable it with either the following ways:
+
+- use uuid that is **generated by your LB application** by setting [`defaultFn: uuid*`](https://loopback.io/doc/en/lb4/Model.html#property-decorator):
```
Comment (dhmlau) [inline README.md:742]: was it an extra `*` after `uuid`? 
```suggestion
- use uuid that is **generated by your LB application** by setting [`defaultFn: uuid`](https://loopback.io/doc/en/lb4/Model.html#property-decorator):
```

File: README.md [hunk 2]
```diff
@@ -724,7 +724,7 @@ export class Order extends Entity {
 Removing or updating the value of `foreignKeys` will be updated or delete or update the constraints in the db tables. If there is a reference to an object being deleted then the `DELETE` will fail. Likewise if there is a create with an invalid FK id then the `POST` will fail.
 " %}
 
-### Auto-generated id property
+### Auto-generated ids
```
Comment (dhmlau) [inline README.md:727]: Just curious.. why are we renaming to `ids`? This section is applicable as long as `id:true`, right?

Comment (agnes512) [reply README.md:727]: Because I used `Auto-generated ids` in MySQL 🙈  I am good with either tho.

Comment (dhmlau) [reply README.md:727]: either way is fine for me. It's good to be consistent as well. Thanks.

**Pull request merged**